starter-code/scripts/articles.js

```diff
@@ -37,7 +45,8 @@ ourLocalData.sort(function(curElem, nextElem) {
 ourLocalData.forEach(function(ele) {
   articles.push(new Article(ele));
 });
-
+//Rendering to HTML
 articles.forEach(function(a) {
+  console.log(a);
```

Comment:
using `console.log` during development and troubleshooting is great, but they shouldn't be present in the submissions.